### PR TITLE
 run rustfmt on lubrustc_typeck/check folder

### PR DESCRIPTION
**Pull request by srinivasreddy**

### CODE REVIEW

**Comment by srinivasreddy** [comment]
@nrc  Please take a look

File: src/librustc_typeck/check/_match.rs [hunk 1]
```diff
@@ -29,7 +29,7 @@ use syntax::ptr::P;
 use rustc::hir::{self, PatKind};
 use rustc::hir::print as pprust;
 
-pub struct PatCtxt<'a, 'gcx: 'a+'tcx, 'tcx: 'a> {
+pub struct PatCtxt<'a, 'gcx: 'a + 'tcx, 'tcx: 'a> {
```
Comment (eddyb) [inline src/librustc_typeck/check/_match.rs]: @nikomatsakis Payback time!

File: src/librustc_typeck/check/_match.rs [hunk 2]
```diff
@@ -122,17 +122,20 @@ impl<'a, 'gcx, 'tcx> PatCtxt<'a, 'gcx, 'tcx> {
                     };
 
                     // Note: spacing here is intentional, we want a space before "start" and "end".
-                    span_err!(tcx.sess, span, E0029,
+                    span_err!(tcx.sess,
+                              span,
+                              E0029,
```
Comment (eddyb) [inline src/librustc_typeck/check/_match.rs]: This is bad, the information is meant to be on a single line.

File: src/librustc_typeck/check/callee.rs [hunk 3]
```diff
@@ -127,20 +131,20 @@ impl<'a, 'gcx, 'tcx> FnCtxt<'a, 'gcx, 'tcx> {
...
+                                                                                &closure_ty.sig)
+                                     .0;
+                    self.record_deferred_call_resolution(def_id,
+                                                         Box::new(CallResolution {
+                                                             call_expr: call_expr,
+                                                             callee_expr: callee_expr,
+                                                             adjusted_ty: adjusted_ty,
+                                                             autoderefs: autoderefs,
+                                                             fn_sig: fn_sig.clone(),
+                                                             closure_def_id: def_id,
```
Comment (eddyb) [inline src/librustc_typeck/check/callee.rs]: This is just weird, I don't understand why you would want anything in braces to be indented more than once from its parent.

**Comment by alexcrichton** [comment]
Closing due to inactivity, but feel free to resubmit with a rebase!

**Pull request closed without merging**